index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" type="text/css" href="style-reset.css">
```

Comment:
- [ ] The `style-reset.css` stylesheet should be the first file called, and after the main stylesheet `style.css`, the reason behind this is because of the specificity of the CSS selectors. The reset stylesheet helps to start with a clean canvas to work with, then you use the custom stylesheet to make it look as intended.